src/main/java/org/myrobotlab/service/InMoov.java

```diff
@@ -1249,7 +1258,7 @@ public InMoovHead startHead(String port, String type, Integer headYPin, Integer
       type = Arduino.BOARD_TYPE_MEGA;
     }
 
-    head.arduino.setBoard(type);
+    // FIXME -  !!! =>  cannot do this "here" ??? head.arduino.setBoard(type);
```

Comment:
we shouldn't allow "setBoard" for the type  perhaps remove this method all together...  when you connect to the arduino. i'm pretty sure it publishes why type of board it is..

Reply:
it can't always be trusted ..  I would like it to be external to the InMoov class ..  it would be nice to get away from "over helping" with configuration

Reply:
fixed with instanceof checking